Content/GUI/Quests/QuestsUIState.cs

```diff
@@ -17,28 +26,63 @@ public void Toggle()
 		if (IsVisible)
 		{
 			IsVisible = false;
+			_questDetails.Remove();
 			return;
 		}
 
-		if (Panel is null)
+		if (!HasChild(_questDetails))
 		{
 			RemoveAllChildren();
-			var localizedTexts = new (string key, LocalizedText text)[]
+			Left = StyleDimension.FromPixels(500);
+			_questDetails = new QuestDetailsPanel
+			{
+				Width = StyleDimension.FromPixels(1200),
+				Height = StyleDimension.FromPixels(900),
+				HAlign = 0.5f,
+				VAlign = 0.5f,
+				ViewedQuest = Main.LocalPlayer.GetModPlayer<QuestModPlayer>().GetAllQuests()[0]
```

Comment:
This always crashes if the player has no quests, and should have a failsafe. That, or a player must be guaranteed to have a quest before opening this UI.

Reply:
Fixed this